es/introduction.md

```diff
@@ -48,25 +48,25 @@ El código público está en el centro de las instituciones públicas modernas,
 
 El software público debe por tanto ser:
 
-* transparente
-* responsable
-* entendible por sus constituyentes
+* Transparente.
+* Responsable.
+* Entendible por sus constituyentes.
 
-Debe de reflejar los valores de la sociedad a la que sirve. Por citar un caso: siendo inclusivo y no discriminatorio.
+Debe reflejar los valores de la sociedad a la que sirve. Por citar un caso: siendo inclusivo y no discriminatorio.
 
-La mayoría de los sistemas de software propietario utilizados actualmente por las organizaciones públicas no cumplen estos requisitos. El código público -el software construido para funcionar con y como infraestructura pública, junto con los acuerdos para su producción- sí lo hace.
+La mayoría de los sistemas de software propietario utilizados actualmente por las organizaciones públicas no cumplen estos requisitos. El código público - el software construido para funcionar con y como infraestructura pública, junto con los acuerdos para su producción - sí lo hace.
```

Comment:
Spanish doesn't use those spaces in between: https://es.wikipedia.org/wiki/Raya_(puntuaci%C3%B3n)

Can also be substituted by normal parenthesis.